.github/workflows/dev-build.yml

```diff
@@ -35,24 +35,19 @@ jobs:
       run: composer install
 
     - name: Build for development
-      run: PREFIX=/tmp/release make test-zip
+      run: PREFIX=/tmp/release make release
 
     - name: Get version name
       run: echo "DEV_BUILD_VERSION=$(git describe --tags)" >> $GITHUB_ENV  
 
-    - name: Set version in plugin
-      run: sed -i -r "s/Version:\ [^$]+$/Version:\ ${{ env.DEV_BUILD_VERSION }}/" /tmp/release/onoffice/plugin.php
-
-    - run: cat /tmp/release/onoffice/plugin.php
-
     - name: Zip files
       run: (cd /tmp/release/ && zip -r ./onoffice-${{ env.DEV_BUILD_VERSION}}.zip ./onoffice/)  # To prevent nested paths (like /tmp/release/onoffice/readme.txt) we have to cd into the directory.
 
     - name: Upload zip
       uses: actions/upload-artifact@v3
       with:
         name: onoffice-${{ env.DEV_BUILD_VERSION }}-please-unpack
-        path: /tmp/release/onoffice-${{ env.DEV_BUILD_VERSION }}.zip  # Uploading a .zip leads to a .zip inside a .zip, but uploading the folder takes significantly longer.
```

Comment:
the old version was fine

Reply:
Do you mean the comment? I'm not sure it is self-explanatory, so I tried to provied a bit more context.